### DISCUSSION TITLE
Operational Emissions: define "consequential approach"

### DISCUSSION THREAD

**Discussion by dschien** [Green-Software-Foundation/sci#193] [SCI feedback]
Related to this: Justify mandating marginal emissions rates as a result from a consequential approach

**Comment by dschien** [comment]
Also rephrase "The marginal emissions rate reflects the change in emissions associcated with a change in demand." What change of demand?

**Comment by atg-abhishek** [comment]
cc @Henry-WattTime something to be considered for inclusion in-line?
